src/isomorphic/classic/types/ReactPropTypes.js

```diff
@@ -123,6 +123,12 @@ if (__DEV__) {
   };
 }
 
+function checkPropTypes(propTypes, object, location, componentName, warnOnRepeat) {
+  return checkReactTypeSpec(propTypes, object, location, componentName, null, null, warnOnRepeat);
```

Comment:
Can you remove the `return` here?

Can you confirm that this never returns a value and never throws? That is important because that is what lets us strip it out safely in production.